sdk/spring/azure-spring-boot-samples/azure-spring-cloud-sample-feature-management/src/main/java/com/azure/spring/sample/featuremanagement/FeatureFilterConfigurations.java

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
This file is different from this one: https://github.com/Azure/azure-sdk-for-java/pull/18916/files#diff-a059b1f86e91608b626baf195859bd4a8628928ae7f1945388b1ee122d922c9dL1

Why?